src/swap/SwapScreen.test.tsx

```diff
@@ -8,12 +9,19 @@ import { ErrorMessages } from 'src/app/ErrorMessages'
 import { TRANSACTION_FEES_LEARN_MORE } from 'src/brandingConfig'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
 import { setSwapUserInput } from 'src/swap/slice'
-import SwapScreen from 'src/swap/SwapScreen'
+import { SwapScreenSection } from 'src/swap/SwapScreen'
```

Comment:
Can you also include a single test for SwapScreen component and make sure it renders the top drawer?